packages/samples/mvp-dapp/src/app.controller.ts

```diff
@@ -34,14 +34,27 @@ function createRecordPattern(lock: Script) {
   }
 }
 
-async function getOmnilockModel(lock: Script): Promise<OmnilockModel> {
+const getLock = (address: string) => {
+  if (!address) {
+    throw new BadRequest('invalid address')
+  }
```

Comment:
I think this is not necessary because if the address is null, the helper.parseAddress will throw an exception.

Reply:
the redundant condition checking was removed by https://github.com/ckb-js/kuai/pull/156/commits/5f8c5f125568ca9a77489327e9615b09f83bc218